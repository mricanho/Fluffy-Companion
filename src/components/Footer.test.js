import React from 'react';
import { render, unmountComponentAtNode } from 'react-dom';
import { act } from 'react-dom/test-utils';
import Footer from './Footer';

let container = null;
beforeEach(() => {
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

it('should render a Footer component', () => {
  act(() => {
    render(
      <Footer />,
      container,
    );
  });

  expect(container.innerHTML).toMatchSnapshot();
});
