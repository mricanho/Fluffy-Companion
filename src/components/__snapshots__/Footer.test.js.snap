// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a Footer component 1`] = `"<footer class=\\"footer\\"><div class=\\"content has-text-centered\\"><p><strong>Fluffy Companion</strong> by<a href=\\"https://miguelricano.me\\"> Miguel Rica√±o</a>. The source code is licensed<a href=\\"http://opensource.org/licenses/mit-license.php\\"> MIT</a>. The website content is licensed<a href=\\"http://creativecommons.org/licenses/by-nc-sa/4.0/\\">CC BY NC SA 4.0</a>.</p></div></footer>"`;
